src/components/sections/SociogramPrompts/PromptFieldsTapBehaviour.js

```diff
@@ -101,6 +102,11 @@ const TapBehaviour = ({
     return true;
   };
 
+  const selectedValue = useSelector((state) => getFormValue(state, 'edges.create'));
+
+  const edgeFilters = useSelector((state) => getEdgeFilters(state));
+  const showNetworkFilterWarning = getEdgeFilteringWarning(edgeFilters, [selectedValue]);
```

Comment:
```suggestion
  const showNetworkFilterWarning = useSelector(getShouldShowEdgeFilterWarning)
```

(see below for implementation)

As an aside, when you have `useSelector((state) => getEdgeFilters(state))` or any situation where the parameters of the inner and outer function are the same (or compatible), you can just do `outerFn(innerFn)` and the parameters will be automatically passed. So in this case `useSelector(getEdgeFilters)`.

Reply:
resolved [3422a77](https://github.com/complexdatacollective/Architect/pull/840/commits/3422a77b6b5bc3fa4cfd493966982cf82585eafa)